config/Dockerfiles/rpmbuild/rpmbuild-test.sh

```diff
@@ -59,6 +59,7 @@ rpmbuild -bs /root/rpmbuild/SOURCES/${fed_repo}.spec
 #TODO
 # Should be a fedora-packager-setup command and a kinit. Will also probably require some packages like fedora-packager/python-krbV
 # Build the package into ./results_${fed_repo}/$VERSION/$RELEASE/ and concurrently do a koji build
+kinit  -k -t /home/fedora.keytab $FEDORA_PRINCIPAL
```

Comment:
We need to add a check at top of script to make sure that **FEDORA_PRINCIPAL** is populated